Find a functionally equivalent code:android.util.Patterns.concatGroups:COMMENT
Method Modifier: public      static      final       
Comment:/**
 * Convenience method to take all of the non-null matching groups in a
 * regex Matcher and return them as a concatenated string.
 *
 * @param matcher      The Matcher object from which grouped text will
 * be extracted
 *
 * @return             A String comprising all of the non-null matched
 * groups concatenated together
 */

@@ -3,9 +3,8 @@
     final int numGroups = matcher.groupCount();
     for (int i = 1; i <= numGroups; i++) {
         String s = matcher.group(i);
-        System.err.println("Group(" + i + ") : " + s);
         if (s != null) {
             b.append(s);
         }

